src/conf_mode/system-login.py

```diff
@@ -42,6 +42,11 @@
 
 radius_config_file = "/etc/pam_radius_auth.conf"
 
+# LOGIN_TIMEOUT from /etc/loign.defs minus 10 sec
+MAX_RADIUS_TIMEOUT: int = 50
+# MAX_RADIUS_TIMEOUT divided by 2 sec (minimum recomended timeout)
+MAX_RADIUS_COUNT: int = 25
```

Comment:
I think 25 RADIUS servers are far to pessimistic.

```
#  If multiple RADIUS server lines exist, they are tried in order.  The
#  first server to return success or failure causes the module to return
#  success or failure.  Only if a server fails to response is it skipped,
#  and the next server in turn is used.
```

IMHO we should limit this maybe to 8 which is more then sufficient (4 IPv4 / 4 IPv6). I have not seen an installation with more then 3 servers ;)

Reply:
> 

Done